airbyte-integrations/bases/base-java/Dockerfile

```diff
@@ -6,6 +6,9 @@ RUN yum install -y tar openssl && yum clean all
 
 WORKDIR /airbyte
 
+# Add the Datadog Java APM agent
+ADD https://dtdg.co/latest-java-tracer dd-java-agent.jar
```

Comment:
Does this mean that we are using the in-process collector or will we be forwarding metrics to the DD collector pod?